packages/core/src/utils/FetchUtil.ts

```
@@ -94,6 +97,9 @@ export class FetchUtil {
         }
       })
     }
+    if (this.clientId) {
```

@tomiir Is the `clientId` required param? Should we return any error if it's not set correctly?

---

No, it's only set on WC relay connections so it might not be set